### PR TITLE
update the output of quota example

### PR DESCRIPTION
**Pull request by guangxuli**
when I learn the project-quota example , I found some outputs should be updated.

### CODE REVIEW

**Comment by deads2k** [comment]
Thanks for the update.  As a separate piece of work, do you feel confident in trying to make the example use the `oc create quota <quota-name> --hard <resource>=<max>` command (recently merged into master)?  I think the shortcut command instead of yaml may make the doc more approachable.

File: examples/project-quota/README.md [hunk 1]
```diff
@@ -109,20 +114,24 @@ $ cat pod-without-resources.yaml
...
     securityContext:
       capabilities: {}
       privileged: false
+    terminationMessagePath: /dev/termination-log
   dnsPolicy: ClusterFirst
   restartPolicy: Always
+status: {} 
 $ oc create -f pod-without-resources.yaml
-Error from server: Pod "pod-without-resources" is forbidden: Limited to 750Mi memory, but pod has no specified memory limit
+Error from server: error when creating "pod-without-resources.yaml": pods "pod-without-resources" is forbidden: Failed quota: quota: must specify cpu,memory
```
Comment (deads2k) [inline examples/project-quota/README.md]: @derekwaynecarr This message got uglified.

**Comment by guangxuli** [comment]
@deads2k thanks, do you mean using " oc create quota quota --hard=cpu=1,memory=750Mi,pods=10,replicationcontrollers=10,resourcequotas=1,services=1"?, I have modifed the description.

**Comment by guangxuli** [comment]
@deads2k @bparees ptal, thanks.

File: examples/project-quota/README.md (hunk 1 above)
Comment (derekwaynecarr) [inline examples/project-quota/README.md]: There is a stutter if the quota name is quota.  I will look at what I can do to improve it upstream.

**Comment by derekwaynecarr** [comment]
LGTM

**Comment by derekwaynecarr** [comment]
[merge]

**Pull request merged**